README.adoc

```diff
@@ -329,9 +342,9 @@ system                 latest              e8a8393e9364        2 minutes ago
 
 === Accessing the internal registry
 
-To run the microservice on the OKD cluster, you need to push the microservice image into a container image registry. You will use OpenShift’s 
-integrated container image registry called OpenShift Container Registry (OCR). First, you must authenticate your Docker client to your OCR. Start by running 
-the login command:
+To run the microservice on the OKD cluster, you need to push the microservice image into a container image registry.
+You will use OpenShift’s integrated container image registry called OpenShift Container Registry (OCR).
```

Comment:
Avoid possessive for with products/inanimate objects. Suggested edit:

You will use OpenShift’s integrated container image registry
--->
You will use the OpenShift integrated container image registry